site/components/Navigation/NavItemMobile.tsx

```diff
@@ -1,26 +1,39 @@
-import Link from "next/link";
-import React, { FC } from "react";
+import React, { Dispatch, FC, SetStateAction, ReactElement } from "react";
 import * as styles from "./styles";
-
+import { cx } from "@emotion/css";
+import { ButtonPrimary } from "@klimadao/lib/components";
+import { NavItem } from "./index";
 interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   name: string;
+  id: NavItem;
   url: string;
-  number?: string;
+  selected: string | undefined;
```

Comment:
As mentioned above, I would rename it the state handling in Parent component to `toggledNavItemID ` 

Then you can pass here the prop: `toggledNavItemID: string` 

and to the check `toggledNavItemID === id`

Reply:
name updated